### DISCUSSION TITLE
Require merge queue - beta

### DISCUSSION THREAD

**Discussion by etienne-mobsuccess** [community/community#47525] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

Hello,

We activate the Require merge queue - beta on our repository : it's a feature that will improve our velocity for sure !
But it does not seems to works for us : if the repository have some required actions on the branch, the PR is added on the queue, but the actions are never executed. After the configured timeout, the PR is removed from the merge queue and not merged.

We don't know if there something else to activate to let it works and if yes, give us the hint, we are really interested with this new feature !!!

Kind regards

**Comment by unknowVariable** [comment]
I'm sorry to hear that you're experiencing issues with the Require merge queue feature on your repository. There could be a few reasons why the required actions are not being executed. Here are a few things you can check:

Verify that the required actions are properly set up on the branch and that they are working correctly when triggered manually.
Make sure that the branch protection rules are set up correctly and that the required actions are listed in the "Required status checks" section.
Check if the GitHub App that powers the merge queue is installed on your repository and has the necessary permissions to trigger the required actions.
If the required actions use third-party services, make sure that the necessary authorization and access tokens are properly set up and have not expired.
If you have verified the above and are still experiencing issues, I recommend reaching out to GitHub Support for further assistance. They may be able to provide additional troubleshooting steps or escalate the issue to the appropriate team for resolution.

I hope this helps and that you are able to get the feature working as intended on your repository.
